### PR TITLE
PCC01 vipinreyo

### PR DESCRIPTION
**Pull request by vipinreyo**
Difficulty level (1-10): [3]
Estimated time spent (hours): [1]
Completed (yes/no): [Yes]
I stretched my coding skills (if yes what did you learn?): [Helped to refresh the comprehensions.]
Other feedback (what can we improve?): [Nope, all good.]

### CODE REVIEW

File: 01/vipinreyo/wordvalue.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+def load_words():
+    """Load dictionary into a list and return list"""
+    with open(DICTIONARY, 'r') as df:
+        return df.read().split()
+
+
+def calc_word_value(word):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    return sum([LETTER_SCORES[l.upper()] for l in word if l in string.ascii_letters])
```
Comment (pybites) [inline 01/vipinreyo/wordvalue.py]: I would break this up in 2 lines (pep8)

File: 01/vipinreyo/wordvalue.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+def max_word_value(words=None):
+    """Calculate the word with the max value, can receive a list
+    of words as arg, if none provided uses default DICTIONARY"""
+
+    if not words:
```
Comment (pybites) [inline 01/vipinreyo/wordvalue.py]: good, or `if words is None`

File: 01/vipinreyo/wordvalue.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        words = load_words()
+
+    word_values = [(word, calc_word_value(word)) for word in words]
+    return sorted(word_values, key=lambda x: x[1], reverse=True)[0][0]
```
Comment (pybites) [inline 01/vipinreyo/wordvalue.py]: this seems more code than needed? what about using `max`?

**Comment by pybites** [comment]
good work

File: 01/vipinreyo/wordvalue.py (hunk 1 above)
Comment (vipinreyo) [reply 01/vipinreyo/wordvalue.py]: Ok, got it. PEP 8 it is.

File: 01/vipinreyo/wordvalue.py (hunk 2 above)
Comment (vipinreyo) [reply 01/vipinreyo/wordvalue.py]: Yes, that's correct.

File: 01/vipinreyo/wordvalue.py (hunk 3 above)
Comment (vipinreyo) [reply 01/vipinreyo/wordvalue.py]: Yes, using max is pythonic.

**Pull request merged**